ui/src/components/common/Breadcrumbs/index.tsx

```diff
@@ -1,61 +1,125 @@
-import React, { useMemo } from "react";
+import React, { useMemo, useContext } from "react";
 import { Link, useLocation } from "react-router-dom";
 import MUIBreadcrumbs from "@mui/material/Breadcrumbs";
 import Typography from "@mui/material/Typography";
+import { AppContext } from "../../../App";
+import chevron from "../../../images/chevron-sm-right.png";
 
 import "./style.css";
 
 export function Breadcrumbs() {
   const location = useLocation();
+  const { systemInfo } = useContext(AppContext);
```

Comment:
assign type `:any` or `:AppContextProps`

Reply:
Updated to proper type.